app/src/main/java/com/kickstarter/ui/helpers/ManagePledgeMenuOptionsFactory.kt

```diff
@@ -19,12 +19,25 @@ fun createManagePledgeMenuOptions(
     ffClient: FeatureFlagClientType
 ): ManagePledgeMenuOptions {
     val backing = project.backing()
-
     val isBackingStatusPreAuth = backing?.status() == Backing.STATUS_PREAUTH
 
-    val isEditPledgeAllowed = ffClient.getBoolean(FlagKey.ANDROID_PLOT_EDIT_PLEDGE)
+    val isFeatureFlagOn = ffClient.getBoolean(FlagKey.ANDROID_PLOT_EDIT_PLEDGE)
+    val isPlotProject = project.isPledgeOverTimeAllowed() == true // If the project allows pledge over time, it is considered a plot project.
+    val isPledgeOverTime = !backing?.paymentIncrements.isNullOrEmpty() // If the backing has payment increments, it is considered a pledge over time.
+    val showEditPledge = when {
+        isFeatureFlagOn && (isPlotProject || isPledgeOverTime) -> project.isLive && !isBackingStatusPreAuth
+        else -> false
+    }
+
+    val showChooseAnotherReward = when {
+        !isFeatureFlagOn -> !isPledgeOverTime || !isPlotProject
+        isFeatureFlagOn -> !(isPlotProject || isPledgeOverTime)
+        else -> false
+    }
```

Comment:
I believe we can simplify the validation logic a bit. Here's an alternative suggestion:

```suggestion
    val showChooseAnotherReward = when {
        !isFeatureFlagOn -> !isPledgeOverTime
        isFeatureFlagOn -> !isPlotProject
        else -> false
    }
```

**Reasoning:**

- When the feature flag is **OFF**, we want to hide the "Choose another reward" option only for **PLOT pledges**.
- When the feature flag is **ON**, we want to hide the option only for **PLOT projects**.

This logic helps keep the conditions more readable while preserving the intended behavior.

Let me know if that makes sense or if there’s any edge case I might have missed!